assets/app/models/Document.js

```diff
@@ -2,27 +2,68 @@ var _ = require('underscore');
 var Backbone = require('backbone');
 var yaml = require('yamljs');
 
+var fileTypes = {
+  'md': ['md', 'markdown'],
+  'yml': ['yml', 'yaml']
+};
+
+var yamlDefaults = [
```

Comment:
I think these defaults are only here as a fallback. On line 27 we initialize the document with opts.content in the file is a yml file. We only use these if no content is provided. Do you think it should be changed to an empty string?